ocaml/xapi/xapi_message.ml

```diff
@@ -406,7 +407,7 @@ let create ~__context ~name ~priority ~cls ~obj_uuid ~body =
   let message = {API.message_name=name;
                  API.message_uuid=uuid;
                  API.message_priority=priority;
-                 API.message_cls=cls;
+                 API.message_cls=cls |> class_to_string |> string_to_class;
```

Comment:
Is this just to make it fail if it is ` ``unknown`?

Reply:
yes